### PR TITLE
Update grsecurity kernels to 4.14.175

### LINKED ISSUE #4989: Upgrade SecureDrop kernels to 4.14.154
## Description

Update SecureDrop kernels to 4.14.154 to mitigate CVE-2019-11135, as well as other vulnerabilities disclosed in Intel network and graphics drivers, detailed in https://blogs.intel.com/technology/2019/11/ipas-november-2019-intel-platform-update-ipu

Also disable `CONFIG_IRDA` in the Kernel configuration per the regression introduced in #4962

### LINKED ISSUE #5111: Update kernels to 4.14.169 or later
## Description

Updating will provide improve cache handling to mitigate CVE-2020-0549 (https://cacheoutattack.com/)
A microcode update will be provided by Intel at a later date.

In order to exploit this vulnerability, an attacker requires:
1. Local code execution on the server
2. A CPU that supports Intel TSX

We do disable TSX in the kernel config: https://github.com/freedomofpress/ansible-role-grsecurity-build/blob/master/files/config-securedrop-4.14#L626 , so this vulnerability may not even be exploitable by an attacker. 

Since the likelihood of exploitation is extremely unlikely. I propose we update Kernels as part of the 1.3.0 release.

### PR DESCRIPTION
**Pull request by conorsch**
Closes #5111

## Status

Work  in progress

## Description of Changes

Fixes #4989, towards #4992 upgrades SecureDrop kernels to 4.14.184

## Testing
- [x] Review and merge https://github.com/freedomofpress/ansible-role-grsecurity-build/pull/57
- [x] Review and merge https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/37
- [x] Restart full CI run for this PR https://circleci.com/workflow-run/1449b8c6-d239-432b-bff0-7b05b50f37fe
- [x] Review and merge this PR

## Deployment

New and existing installs will be updated via deb packages and unattended upgrades via cron-apt

### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

### If you made changes to documentation:

- [ ] Doc linting (`make docs-lint`) passed locally

### If you added or updated a code dependency:

Choose one of the following:

- [ ] I have performed a diff review and pasted the contents to [the packaging wiki](https://github.com/freedomofpress/securedrop-debian-packaging/wiki)
- [ ] I would like someone else to do the diff review

### CODE REVIEW

**Comment by zenmonkeykstop** [comment]
4.14.175 kernels good on Dell r620 and r440 test hardware, no install issues or errors on first reboot. Will take a look again after they've been running for a nightly reboot cycle.

File: install_files/ansible-base/group_vars/all/securedrop [hunk 1]
```diff
@@ -40,5 +40,5 @@ securedrop_cond_reboot_file: /tmp/sd-reboot-now
 
 # If you bump this, also remember to bump in molecule/builder/tests/vars.yml
 securedrop_pkg_grsec:
-  ver: "4.14.154"
+  ver: "4.14.175"
   depends: "linux-image-4.14.154-grsec-securedrop,linux-image-4.4.182-grsec,linux-firmware-image-4.4.182-grsec,intel-microcode"
```
Comment (conorsch) [inline install_files/ansible-base/group_vars/all/securedrop:44]: The `depends:` line should be updated with the new version package. Overlooked that in my first round of changes. Will amend, build, update https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/37, then add the test changes suggested by @emkll.

Comment (conorsch) [reply install_files/ansible-base/group_vars/all/securedrop]: Done. Ready for re-review.

File: install_files/ansible-base/group_vars/all/securedrop [hunk 2]
```diff
@@ -38,7 +38,7 @@ enable_ssh_over_tor: true
 # the many roles of the current prod playbook
 securedrop_cond_reboot_file: /tmp/sd-reboot-now
 
-# If you bump this, also remember to bump in molecule/builder/tests/vars.yml
+# If you bump this, also remember to bump in molecule/builder-xenial/tests/vars.yml
 securedrop_pkg_grsec:
-  ver: "4.14.154"
-  depends: "linux-image-4.14.154-grsec-securedrop,linux-image-4.4.182-grsec,linux-firmware-image-4.4.182-grsec,intel-microcode"
+  ver: "4.14.175"
+  depends: "linux-image-4.14.175-grsec-securedrop,linux-image-4.4.182-grsec,linux-firmware-image-4.4.182-grsec,intel-microcode"
```
Comment (emkll) [inline install_files/ansible-base/group_vars/all/securedrop:44]: is there a reason we are still dependending on an older 4.4 series kernel, and not the last known good kernel (4.14.154)? should we not simply preserve the previous known good kernel version

Comment (conorsch) [reply install_files/ansible-base/group_vars/all/securedrop:44]: Great point, @emkll. I admit the same issue gave my pause while making the update. Will raise in standup today to make sure support team doesn't have any concerns, then proceed with updating to current-and-previous as standard going forward.

Comment (conorsch) [reply install_files/ansible-base/group_vars/all/securedrop:44]: No concerns raised in standup, proceeding with updating the pinned dependencies as suggested.

**Comment by emkll** [comment]
Thanks @conorsch diff looks good visually. Based on my review and @zenmonkeykstop 's review above, I think we can now update the metapackage in https://github.com/freedomofpress/securedrop-dev-packages-lfs/pull/37 and proceed to the next step in the PR's testplan.

**Comment by emkll** [comment]
CI appears to be failing due to the latest Tor Browser update (https://circleci.com/gh/freedomofpress/securedrop/39518?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) 

@conorsch since the merge of this PR is blocking other PRs (due to merge of the kernels to the pat repo), would you mind appending a commit to this branch (similar to https://github.com/freedomofpress/securedrop/pull/5173/files)? The latest version is 9.0.9 (https://www.torproject.org/download/)

**Comment by conorsch** [comment]
Oops, just seeing your message @emkll, jumping on the TBB fix to unbreak CI...

**Pull request merged**